subprojects/veins_inet3/src/veins_inet/VeinsInetMobility.cc

```diff
@@ -32,8 +32,6 @@
 
 namespace veins {
 
-using inet::Coord;
```

Comment:
Any particular reason for changing this (beyond personal preference)? It does not seem to be required to achieve the goals mentioned in the commit message

Reply:
Including the traci references:
```
#include "veins/modules/mobility/traci/TraCIScenarioManager.h"
#include "veins/modules/mobility/traci/TraCICommandInterface.h"
```
Gives out this error:
```
./veins_inet/VeinsInetMobility.h:44:13: error: ‘Coord’ is already declared in this scope
```
This is the reason for this modification. I think someone else had this problem before because the same change could be found in:
[subprojects/veins_inet/src/veins_inet/VeinsInetMobility.h](https://github.com/sommer/veins/blob/master/subprojects/veins_inet/src/veins_inet/VeinsInetMobility.h)